mobile/src/navigation/app/components/become-mentor/become-mentor.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+import saly from '~/assets/images/saly.png';
+import { DataStatus } from '~/common/enums/enums';
+import { Button, Image, Spinner, View } from '~/components/common/common';
+import { getImageUri } from '~/helpers/helpers';
+import { useAppSelector } from '~/hooks/hooks';
+
+import { styles } from './styles';
+
+type Props = {
+  onPress: () => void;
+};
+
+const BecomeMentor: FC<Props> = ({ onPress }) => {
+  const dataStatus = useAppSelector(
+    ({ courses }) => courses.dataBecomeMentorStatus,
+  );
+
+  if (dataStatus == DataStatus.PENDING) {
```

Comment:
Make sure it will be displayed correctly

Reply:
Oh....
stupid mistake